lib/ex_aws/utils.ex

```diff
@@ -100,4 +100,6 @@ defmodule ExAws.Utils do
 
     greg_secs - @seconds_0_to_1970
   end
+
+  def uuid, do: DateTime.utc_now |> :erlang.phash2
```

Comment:
I'm gonna switch this to `:erlang.unique_integer([:positive])` since concurrent requests could easily call this function at the same time.